src/pocketmine/Player.php

```diff
@@ -2477,6 +2479,9 @@ public function handleInventoryTransaction(InventoryTransactionPacket $packet) :
 
 				switch($type){
 					case InventoryTransactionPacket::USE_ITEM_ON_ENTITY_ACTION_INTERACT:
+						if($target instanceof Villager && $target->isCanTrade()){
```

Comment:
`isCanTrade()` doesn't sound like correct English - `canTrade()` would be sufficient I think.